readme.md

```diff
@@ -261,7 +261,33 @@
             </table>            
         </td>
     </tr>
-    
-    
-</table>
-<!--// CONTRIBUTER //-->
```

Comment:
you should not remove these two lines, please return them back and add your `<tr>...</tr>` inside of them